src/geant4/generate_geant4

```diff
@@ -2,9 +2,11 @@
 
 from optparse import OptionParser
 import os,errno
-from itaps import iMesh,iBase
+import sys
+import meshio
```

Comment:
I think we should convert the `itaps` code to use the `pyne.dagmc` class rather than the `meshio` module